personalities/sctfe/ct_server_gcp/main.go

```diff
@@ -276,3 +284,23 @@ func newGCPStorage(ctx context.Context, signer note.Signer) (*sctfe.CTStorage, e
 
 	return sctfe.NewCTSTorage(tesseraStorage, issuerStorage, beDedupStorage)
 }
+
+type timestampFlag struct {
+	t *time.Time
+}
+
+func (t *timestampFlag) String() string {
+	if t.t != nil {
+		return t.t.Format(time.RFC3339)
+	}
+	return ""
+}
+
+func (t *timestampFlag) Set(w string) error {
```

Comment:
Would it be a good idea to require that these timestamps are specified in UTC?

Reply:
How would you do this in practice? Manually parse the string to check that no zone other than "Z" is specified?

Reply:
Good question :)

Some ideas/thoughts: you could make the flag names be `..._utc` to make it clear what they should be, and then either:
- Check for strings.HasSuffix("Z"), or
- Specify "2006-01-02T15:04:05" as the layout for the call to `Parse`
?

Probably not a good idea to try to use `Time.Location()` to figure anything out after the fact, though.

Reply:
Add a check to require UTC, i.e "Z" suffix. Temporal logs are not formally defined anywhere, so nothing mandates that they all end on an UTC date. In practice, all of them do I think.  This check should incentive to stick to this, but we can't really force folks to do so.  I'm a bit reluctant to specific a layout without a "Z", time formatting is already enough of a mess :)